components/commentsList.js

```diff
@@ -4,14 +4,23 @@
 		const ctrl = this;
 
 		ctrl.comments = [{text: "Nice...", isRemoved: false}, 
-	    				   {text:"Dumb", isRemoved: false}, 
+	    				   {text:"Dumb", isRemoved: false},
 	    				   {text: "Dope", isRemoved: false},
 	    				   {text: "Brilliant", isRemoved: false},
 	    				   {text: "Generic", isRemoved: false},
 	    				   {text: "Outstanding", isRemoved: false},
 	    				   {text: "Borring", isRemoved: false}];
+
```

Comment:
create commentsList array. This array equal the comments array on init your app and this array(commentsList) use in your template